panic in the unit-tests

```
+ go test -test.timeout=30m github.com/docker/docker/pkg/httputils
--- FAIL: TestResumableRequestReader (0.00s)
panic: httptest: failed to listen on a port: listen tcp6 [::1]:0: bind: cannot assign requested address [recovered]
	panic: httptest: failed to listen on a port: listen tcp6 [::1]:0: bind: cannot assign requested address

goroutine 7 [running]:
testing.func·006()
	/usr/local/go/src/testing/testing.go:441 +0x181
net/http/httptest.newLocalListener(0x0, 0x0)
	/usr/local/go/src/net/http/httptest/server.go:68 +0x457
net/http/httptest.NewUnstartedServer(0x7fd6135fa110, 0xc20802ad80, 0x7fd6135fa110)
	/usr/local/go/src/net/http/httptest/server.go:96 +0x1f
net/http/httptest.NewServer(0x7fd6135fa110, 0xc20802ad80, 0xac3578)
	/usr/local/go/src/net/http/httptest/server.go:83 +0x32
github.com/docker/docker/pkg/httputils.TestResumableRequestReader(0xc20800c120)
	/go/src/github.com/docker/docker/pkg/httputils/resumablerequestreader_test.go:18 +0xe5
testing.tRunner(0xc20800c120, 0xa9d9a0)
	/usr/local/go/src/testing/testing.go:447 +0xbf
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:555 +0xa8b

goroutine 1 [chan receive]:
testing.RunTests(0x7c25a0, 0xa9d9a0, 0x2, 0x2, 0x100000001)
	/usr/local/go/src/testing/testing.go:556 +0xad6
testing.(*M).Run(0xc20802e460, 0xac2e40)
	/usr/local/go/src/testing/testing.go:485 +0x6c
main.main()
	github.com/docker/docker/pkg/httputils/_test/_testmain.go:102 +0x291

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35

Tests failed: ./pkg/httputils

+ go test -test.timeout=30m github.com/docker/docker/pkg/ioutils
PASS
coverage: 79.6% of statements

+ go test -test.timeout=30m github.com/docker/docker/
```

--- a/hack/vendor.sh
+++ b/hack/vendor.sh
@@ -55,7 +55,7 @@ clone hg code.google.com/p/go.net 84a4013f96e0
 clone hg code.google.com/p/gosqlite 74691fb6f837
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork 67438080724b17b641b411322822c00d0d3c3201
+clone git github.com/docker/libnetwork 2da2dc055de5a474c8540871ad88a48213b0994f
 clone git github.com/vishvananda/netns 008d17ae001344769b031375bdb38a86219154c6
 clone git github.com/vishvananda/netlink 8eb64238879fed52fd51c5b30ad20b928fb4c36c
 

